src/App.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
-import * as React from "react";
-import { Parser } from "html-to-react";
-import AppSearchAPIConnector from "@elastic/search-ui-app-search-connector";
+import * as React from 'react';
+import { Parser } from 'html-to-react';
+import AppSearchAPIConnector from '@elastic/search-ui-app-search-connector';
```

Comment:
Please don't format the code as it makes reviews difficult.

Reply:
noted